lms/views/lti/basic_launch.py

```diff
@@ -52,6 +52,11 @@ def __init__(self, context, request) -> None:
             self.request.lti_user.application_instance, self.request.lti_params
         )
         self.course = self._record_course()
+        # Keep a record of every the user in the DB
```

Comment:
I guess there's a typo in this sentence, but I'm not sure what you were trying to say.